docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.1'
+services:
+  rekolekcje-engine:
+    container_name: reko-engine
+    build: ./rekolekcje-engine/
+    ports:
+      - "5000:5000"
+    links:
+      - rekolekcje-db:database
+    networks:
+      - reko_network
+    environment:
+      - spring.datasource.url=jdbc:postgresql://rekolekcje-db:5432/rekolekcjedb
```

Comment:
czy rekolekcjedb nie mogłoby być tutaj wyznaczane na podstawie POSTGRES_DB? Chodzi mi o to, że jest duplikacja i trzeba tę nazwę zmieniać w dwóch miejscach. Nie znam się za bardzo na docker-compose, ale pewnie można to jakoś zrobić, żeby wyciągać ze zmiennej zamiast podawać explicite.